src/mattack_actors.cpp

```diff
@@ -304,7 +304,7 @@ bool gun_actor::call( monster &z ) const
     if( z.friendly ) {
         int max_range = 0;
         for( const auto &e : ranges ) {
-            max_range = std::max( { max_range, e.first.first, e.first.second } );
+            max_range = std::max( std::max( max_range, e.first.first ), e.first.second );
```

Comment:
I'm torn on that. It's not a lightweight header but I suspect we have a build error waiting to happen there given the right combination of platform and compiler.